mp/src/game/shared/SoundEmitterSystem.cpp

```diff
@@ -311,6 +313,7 @@ class CSoundEmitterSystem : public CBaseGameSystem
 			}
 		}
 #endif
+		pHostTimescale = cvar->FindVar( "host_timescale" );
```

Comment:
This can instead be a ConVarRef inside the system.

Reply:
I was going to do this but it seems way more cumbersome as I'd have to initialize it more than once.